core/services/relay/evm/chain_reader_test.go

```diff
@@ -31,6 +33,8 @@ import (
 )
 
 const commonGasLimitOnEvms = uint64(4712388)
+const chainReaderContractName = "LatestValueHolder"
+const returnSeenName = "ReturnSeen"
 
```

Comment:
Same goes for codec interface tester,  are they supposed to implement BasicTester interface from chainlink-common/pkg/types/interfacetests/utils.go? If so it would be nice to include `var _ BasicTester = chainReaderInterfaceTester{}` and var _ BasicTester = codecInterfaceTester{} to make it obvious

Reply:
They implement CodecInterfaceTester and ChainReaderInterfaceTester.  Those each implement the BasicTester.  It's required implicitly in the same file because running the test takes their respective tester.  I figured that since it's used in the same file, and you'll get the same error if you do var _ and if you don't there wasn't a reason to add it.